app/src/main/java/com/rockspin/androiddevtest/MyApplication.java

```diff
@@ -5,12 +5,23 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
+import java.util.List;
+
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
 
 public class MyApplication extends Application {
 
     private static APIService sAPIService;
+    private static MyApplication s_instance;
+    private List<CosmonautActivity> m_cosmonautActivities;
```

Comment:
Sorry I posted this in error. What I mean in that the list of cosmonautsactivities is effectively a static global variable accessible from anywhere in the application.

If you moved it to the activity it would be better encapsulated. It's a small point